extension/scripts/global/features.js

```diff
@@ -0,0 +1,307 @@
+"use strict";
+
+/*
+ * Add prototype functions.
+ */
+
+// TODO - Use in ttItems.js
+const ITEM_VALUE_UTILITIES = {
```

Comment:
The file is global/features.js but the 2 main objects are called _UTILITIES and _DETAILS? Could be something to look at in the future

Reply:
I temporary renamed the file to `features.js`, because those 2 are (parts of) a feature.

`ITEM_VALUE_UTILITIES` has some utilities for the Item Value feature, and a version of the Item Value feature that works on some pages.
`DRUG_DETAILS` is the entire Drug Details feature.

These will be moved in one of the upcoming PRs.